src/core/AutomationPattern.cpp

```diff
@@ -72,6 +80,14 @@ AutomationPattern::AutomationPattern( const AutomationPattern & _pat_to_copy ) :
 		m_timeMap[it.key()] = it.value();
 		m_tangents[it.key()] = _pat_to_copy.m_tangents[it.key()];
 	}
+	if( getTrack()->trackContainer()->type() == TrackContainer::BBTrackContainer )
```

Comment:
Since this is an enum, is this a good `select/case` candidate (for readiblity of course)?